src/lib/containers/GenericCard.tsx

```diff
@@ -207,39 +223,86 @@ export const renderLabel = (args: {
       )
     } else {
       // they don't need a link
-      return str
+      return <>{str}</>
     }
   }
 
   if (labelLink.isMarkdown) {
     if (strList) {
-      return strList.map((el, index) => {
-        return (
-          <span key={el}>
-            <MarkdownSynapse key={el} renderInline={true} markdown={el} />
-            {/* \u00a0 is a nbsp; */}
-            {index < strList.length - 1 && ',\u00a0\u00a0'}
-          </span>
-        )
-      })
+      return (
+        <>
+          {strList.map((el, index) => {
+            return (
+              <span key={el}>
+                <MarkdownSynapse key={el} renderInline={true} markdown={el} />
+                {/* \u00a0 is a nbsp; */}
+                {index < strList.length - 1 && ',\u00a0\u00a0'}
+              </span>
+            )
+          })}
+        </>
+      )
     } else {
       return <MarkdownSynapse renderInline={true} markdown={value} />
     }
   }
   const split = strList ? strList : str.split(',')
-  return split.map((el, index) => {
-    const { baseURL, URLColumnName, wrapValueWithParens } = labelLink
-    const value = wrapValueWithParens ? `(${el})` : el
-    const href = `/${baseURL}?${URLColumnName}=${value}`
+  if ('linkColumnName' in labelLink) {
+    const linkIndex =
+      selectColumns?.findIndex(el => el.name === labelLink.linkColumnName) ||
+      columnModels?.findIndex(el => el.name === labelLink.linkColumnName)
+    if (linkIndex == null) {
+      console.warn(
+        `Could not determine column index of ${labelLink.linkColumnName}`,
+      )
+      return <>{value}</>
+    } else if (row == null) {
+      // Sorry future dev, I couldn't figure out how to pass the row down from GenericCard because it's not fully typed
+      console.warn(
+        `Couldn't create link because row had value ${row}. Specifying a link via column is not currently possible in cards`,
+      )
+      return <>{value}</>
```

Comment:
Handling common failure cases when trying to call this from a card instead of table cell